spring-cloud-app-broker-docs/src/docs/asciidoc/service-instances.adoc

```diff
@@ -212,6 +212,53 @@ If you use the `SpacePerServiceInstance` target, App Broker will deploy backing
 
 If you use the `ServiceInstanceGuidSuffix` target, App Broker will deploy backing applications using a unique name and hostname that incorporates the service instance GUID provided by the platform at service instance create time. For Cloud Foundry, the target location will be the org named by `spring.cloud.appbroker.deployer.cloudfoundry.default-org`, the space named by `spring.cloud.appbroker.deployer.cloudfoundry.default-space`, and an application name as `[APP-NAME]-[SI-GUID]`, where `[APP-NAME]` is the `name` listed for the application under `spring.cloud.appbroker.services.apps` and `[SI-GUID]` is the service instance GUID. The application will also use a hostname incorporating the service instance GUID as a suffix, as `[APP-NAME]-[SI-GUID]`.
 
+====== Creating a custom Target
+
+If you want to create a custom Target, app broker provides a flexible way to add new targets by  custom one by just creating a new `Bean` that extends from `TargetFactory` and implementing the `create` method:
```

Comment:
I think "by  custom one" here is a typo?

Reply:
Good catch! Just changed it.